security/pfSense-pkg-snort/files/usr/local/pkg/snort/snort_check_for_rule_updates.php

```diff
@@ -377,6 +377,20 @@ function snort_fetch_new_rules($file_url, $file_dst, $file_md5, $desc = "") {
 	}
 }
 
+function snort_untar($mode, $tarFile, $outputFolder = null, $extra = null){
+	$cmd = "/usr/bin/tar {$mode} {$tarFile}";
+	if($outputFolder != null) $cmd .= " -C {$outputFolder}";
+	if($extra != null) $cmd .= " {$extra}";
+	$ret = snort_exec($cmd);
+	return $ret === 0;
+}
+
+function snort_exec($cmd, &$output = null){
+	$return = -1;
+	exec($cmd, $output, $return);
+	return $return;
+}
```

Comment:
This function is not necessary.  exec() call can happen inside snort_untar() without any harm

Reply:
I have created it because I saw that there were many other calls to `exec()` in there, that did not check for success/failure. Maybe you can reuse it in the future. I thought of replacing those calls myself but then the chances of breaking something would be more, since I cannot test the code.

Reply:
But if you are sure I should keep it as one function, I can do that. :)